src/01-dinosaur-facts.js

```diff
@@ -44,7 +69,21 @@ function getLongestDinosaur(dinosaurs) {}
  *  getDinosaurDescription(dinosaurs, "incorrect-id");
  *  //> "A dinosaur with an ID of 'incorrect-id' cannot be found."
  */
-function getDinosaurDescription(dinosaurs, id) {}
+function getDinosaurDescription(dinosaurs, id) {
+    let identifiedDinosaur = dinosaurs.find(dinosaur => dinosaur.dinosaurId == id);
+
+    if(identifiedDinosaur == undefined) {
+      
+      return `A dinosaur with an ID of '${id}' cannot be found.`;
+
+    };
+    
+    return `${identifiedDinosaur.name} (${identifiedDinosaur.pronunciation})\n${identifiedDinosaur.info} It lived in the ${identifiedDinosaur.period} period, over ${identifiedDinosaur.mya.length > 1 ? identifiedDinosaur.mya[1] : identifiedDinosaur.mya[0]} million years ago.`;
```

Comment:
great ternary - this is a perfect example of a chance for a _helper_function.  The code works but becomes hard to read when it's this long.  `return _getDinoDescription()` would be much easier to read